.github/workflows/functional_test.yaml

```diff
@@ -11,14 +11,14 @@ jobs:
       fail-fast: false
       matrix:
         # The kubernetes_version matrix entries are currently following native Kubernetes version support and +1, see: https://endoflife.date/kubernetes
-        # To check latest versions, see: https://hub.docker.com/r/kindest/node/tags
+        # To check latest Minikube versions, see: https://raw.githubusercontent.com/kubernetes/minikube/master/pkg/minikube/constants/constants_kubernetes_versions.go
         # TODO: Automate updating this and expand/contract matrix coverage for other Kubernetes distributions
         kubernetes_version:
-          - v1.29.0 # Support: 28 Dec 2024 - 28 Feb 2025
-          - v1.28.4 # Support: 28 Aug 2024 - 28 Oct 2024
-          - v1.27.8  # Support: 28 Apr 2024 - 28 Jun 2024
-          - v1.26.11 # Support: 28 Dec 2023 - 28 Feb 2024
-          - v1.25.16 # Support: 27 Aug 2023 - 27 Oct 2023
+          - v1.30.0 # Support: 28 Apr 2025 - 28 Jun 2025
+          - v1.29.4 # Support: 28 Dec 2024 - 28 Feb 2025
+          - v1.28.9 # Support: 28 Aug 2024 - 28 Oct 2024
+          - v1.27.13  # Support: 28 Apr 2024 - 28 Jun 2024
+          - v1.26.15 # Support: 28 Dec 2023 - 28 Feb 2024
```

Comment:
If support for this has ended should we still be testing it?

Reply:
We should continue this practice for now. An existing comment above indicates that we test the current native Kubernetes versions plus one out-of-date version, primarily to accommodate distributions like EKS, which supports older versions not covered by native Kubernetes, and we support EKS.

I'll update the comments to highlight the point above.